src/App.scss

```diff
@@ -103,24 +103,14 @@
   }
 }
 
-.apid-operations {
-  .operation-summary {
-    margin-right: 20px;
-  }
-  .operation-path {
-    color: var(--pf-global--Color--dark-200);
-  }
-  .pf-c-accordion__toggle {
-    flex-direction: row-reverse;
-
-    .pf-c-accordion__toggle-text {
-      flex-grow: 1;
-      text-align: left;
-      margin-left: 1.5em;
-    }
-  }
-  .pf-c-accordion__expanded-content-body {
-    padding-left: 4em;
+.apid-c-accordion-operations.pf-c-accordion {
+  --pf-c-accordion__expanded-content-body--PaddingLeft: var(--pf-global--spacer--2xl);
+  border-left: 1px solid var(--pf-global--BorderColor--100);
+  border-right: 1px solid var(--pf-global--BorderColor--100);
+  .pf-c-accordion__toggle-text {
+    flex-grow: 1;
+    text-align: left;
+    margin-left: var(--pf-global--spacer--md);
```

Comment:
![at-least-i-tried](https://user-images.githubusercontent.com/3845764/220409690-c0abe4e1-e437-4171-9f33-5ee3740bab12.png)

Reply:
@josejulio you did great!